src/core/core.helpers.js

```diff
@@ -173,17 +149,16 @@ module.exports = function(Chart) {
 	helpers.getValueOrDefault = function(value, defaultValue) {
 		return value === undefined ? defaultValue : value;
 	};
-	helpers.indexOf = function(arrayToSearch, item) {
-		if (Array.prototype.indexOf) {
-			return arrayToSearch.indexOf(item);
-		} else {
-			for (var i = 0; i < arrayToSearch.length; i++) {
-				if (arrayToSearch[i] === item)
+	helpers.indexOf = Array.prototype.indexOf?
```

Comment:
I don't know exactly what browsers compatibility we want to target. What do you think about these 2 helpers?

**[Array.isArray](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray)**
![image](https://cloud.githubusercontent.com/assets/3874900/15815953/7ba0e448-2bd0-11e6-8871-7dd280b1cffb.png)

**[Array.prototype.indexOf](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)**
![image](https://cloud.githubusercontent.com/assets/3874900/15815979/a19908e2-2bd0-11e6-95e7-8fb4b20606cf.png)